about.html

```diff
@@ -7,59 +7,105 @@
   <title>Sprint Challenge - About</title>
 
   <link href="https://fonts.googleapis.com/css?family=Roboto|Rubik" rel="stylesheet">
-  <link rel="stylesheet" href="css/index.css">
+  <link rel="stylesheet" href="/about.css">
 
 </head>
 
 <body>
-    <div class="container about-page">
-     
-    About
+  <nav class="navRow">
+    <ul>
```

Comment:
If you would like you can simplify this into just a nav a instead of using list elements.